src/sonic-yang-models/yang-models/sonic-fabric-monitor.yang

```diff
@@ -32,17 +32,31 @@ module sonic-fabric-monitor{
                 }
 
                 leaf monPollThreshIsolation {
-                    type uint32;
+                    type uint8;
                     default 1;
                     description "Consecutive polls with higher error rate for isolation.";
                 }
 
                 leaf monPollThreshRecovery {
-                    type uint32;
+                    type uint8;
                     default 8;
                     description "Consecutive polls with lesser error rate for inclusion.";
                 }
 
+                leaf monCapacityThreshWarn {
+                    type uint8;
+                    default 10;
+                    description "Percentage of up fabric links.";
+                }
+
+                leaf monState {
+                    description "Configuration to set fabric link monitoring state: enable/disable";
+                    type string {
```

Comment:
If only accept enable/disable, we can remove length?
And can be used by type stypes:mode-status;
Check example here: https://github.com/sonic-net/sonic-buildimage/blob/cab94964d94381ac399361436e85d6ca6637cbd9/src/sonic-yang-models/yang-models/sonic-interface.yang#L80

Reply:
> sonic-buildimage/src/sonic-yang-models/yang-models/sonic-interface.yang

got it, revised based on your suggestion.